src/PlayAudioUsingMediaStreams/Program.cs

```diff
@@ -4,7 +4,12 @@
 
 // Add services to the container.
 
+builder.Services.AddSpeechClient();
```

Comment:
Adding speech client so it can be injected instead of constructing it ourselves.
Learned about it here: https://cloud.google.com/dotnet/docs/reference/help/client-lifecycle